lang/en.yml

```diff
@@ -75,7 +107,7 @@ en:
   SilverStripe\CMS\Controllers\CMSPageEditController:
     ErrorItemPermissionDenied: "It seems you don't have the necessary permissions to add {ObjectTitle} to a campaign"
     ErrorNotFound: "That {Type} couldn't be found"
-    MENUTITLE: 'Edit Page'
+    MENUTITLE: Pages
```

Comment:
This new value is correct. I have confirmed that in CMS 5 this and the other `MENUTITLE` change don't affect anything (or at least if they do, I wasn't able to identify it)